src/rpm/headers/header.rs

```diff
@@ -123,54 +129,67 @@ where
         self.index_entries
             .iter()
             .find(|entry| &entry.tag == tag)
-            .ok_or_else(|| RPMError::new(&format!("unable to find Tag {}", tag)))
+            .ok_or_else(|| RPMError::TagNotFound(format!("{}", tag)))
```

Comment:
```suggestion
            .ok_or_else(|| RPMError::TagNotFound(tag.to_string()))
```

Reply:
I prefer the the `.to_string()` variants, since it introduces less visual clutter and does the same.

Reply:
yeah, I completely forgot that `to_string` was implied by the implemented traits.